.editorconfig

```diff
@@ -8,6 +8,6 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{package.json,.travis.yml,.eslintrc}]
+[{package.json,bower.json,.travis.yml,.eslintrc}]
```

Comment:
If you add `.json` extension to your `.eslintrc` files (which is recommended anyway), you can simplify this rule to just be `*.json,*.yml`.